htdocs/langs/sl_SI/accountancy.lang

```diff
@@ -178,6 +178,7 @@ ACCOUNTING_SELL_JOURNAL=Sales journal - sales and returns
 ACCOUNTING_PURCHASE_JOURNAL=Purchase journal - purchase and returns
 ACCOUNTING_BANK_JOURNAL=Cash journal - receipts and disbursements
 ACCOUNTING_EXPENSEREPORT_JOURNAL=Pregled stroškovnih poročil
+ACCOUNTING_ACCOUNT_EXPENSEREPORT=Default accounting account for "expense reports"
```

Comment:
You should not add any entry into language files, except en_US.
All other files are generated from transifex when transifex sync is done.
Can you repush a PR with just the en_US change ?

Reply:
Oh, ok
my bad, i will remove it